### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by Kurzawskyy**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+    computerSelectedItem: '',
+    round: 1,
+    points: [
+        {user: 0},
+        {computer: 0}
+    ]
+};
 
+
+// Modal getting elements
```
Comment (Lukasz-Semik) [inline app.js]: in fact, variable `modal` is element itself. 
So - more proper comment would be:
`// Modal elements` (as we are not creating a function, but variable storing actual html element)

File: app.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+const modal = document.getElementById('modal');
+const submitNameBtn = document.getElementById('submit-name-btn');
+const inputName = document.getElementById('first-user-name');
```
Comment (Lukasz-Semik) [inline app.js]: why `id` has `first` ? I guess that we have only `user-name`

File: app.js [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+
+// game getting elements
```
Comment (Lukasz-Semik) [inline app.js]: `Game elements` (let's decide if comments starts with capital letter or lower cased letter)

File: app.js [hunk 4]
```diff
@@ -14,20 +24,54 @@ const choosenPaper = document.getElementById('choosen-paper');
...
+    modal.classList.add('invisible');
+})
+
+// After modal
+function showButton() {
+    submitButton.classList.add('show-button');
+}
+
+function startPlaying() {
+    // Computer's turn
```
Comment (Lukasz-Semik) [inline app.js]: let's add `TODO: `

File: styles.css [hunk 5]
```diff
@@ -112,3 +112,98 @@ header {
...
+    padding: 20px;
+}
+
+.modal-title {
+    height: 30%;
+    width: 96%;
+    margin: auto;
+    text-align: center;
+    font-size: 35px;
+    background-image: linear-gradient(to top, #FC00FF, #00DBDE);
```
Comment (Lukasz-Semik) [inline styles.css:158]: I think that it could be `background: linear-gradient....`
ANd then, I don't think next line is needed

File: styles.css [hunk 6]
```diff
@@ -112,3 +112,98 @@ header {
...
+}
+
+.input {
+    border: 0;
+    border-bottom: 2px solid black;
+    width: 90%;
+    font-size: 20px;
+    outline: none;
+    transition: border-color 0.5s linear;
+    margin-bottom: 20px;
```
Comment (Lukasz-Semik) [inline styles.css]: let's use shorthand:

`margin: 20px 0 20px 46px`

**Review by Lukasz-Semik — COMMENTED** [review]
very goood job! 👏

File: styles.css (hunk 5 above)
Comment (Kurzawskyy) [reply styles.css:158]: It has to be there, bc if there's not it shows only linear background, not text.

File: app.js [hunk 7]
```diff
@@ -31,10 +34,14 @@ const givenName = document.getElementById('user-name');
 
 // Modal running
 submitNameBtn.addEventListener('click', () => {
-    appState.userName = inputName.value;
-    givenName.innerHTML = appState.userName;
-
-    modal.classList.add('invisible');
+    if(inputName.value === '') {
+        alert('You must enter your name there.');
```
Comment (Lukasz-Semik) [inline app.js]: let's keep im mind, that we can handle it better by showing proper error, for example under the input, but it's not for this PR.

**Review by Lukasz-Semik — APPROVED** [review]

File: app.js [hunk 8]
```diff
@@ -34,8 +35,8 @@ const givenName = document.getElementById('user-name');
 
 // Modal running
 submitNameBtn.addEventListener('click', () => {
-    if(inputName.value === '') {
-        alert('You must enter your name there.');
+    if(!inputName.value) {
+        errorMsg.classList.add('item-visible');
```
Comment (Lukasz-Semik) [inline app.js:39]: you have already created an `invisible` class, maybe we could reuse it ? :)

File: styles.css [hunk 9]
```diff
@@ -112,3 +109,106 @@ header {
...
+
+.submit-name-btn:hover {
+    background-color: #5433FF;
+}
+
+.invisible {
+    display: none;
+}
+
+.item-visible {
```
Comment (Lukasz-Semik) [inline styles.css:212]: I guess it should be removed - do you use this class at this moment?

**Review by Lukasz-Semik — APPROVED** [review]

**Pull request merged**